<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Консоль</string>
     </attribute>
     <widget class="QPlainTextEdit" name="consoleEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>745</width>
        <height>475</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Функционал</string>
     </attribute>
     <widget class="QComboBox" name="portsComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="portsLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Выберите COM-порт</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="baudRateLineEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="baudRateLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите Baud Rate </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_connect">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="field_view">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>330</width>
        <height>228</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>228</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>228</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g00">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G00 Быстрое позиционирование</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g01">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G01 Линейная интерполяция</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g02">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G02 Круговая интерполяция по часовой стрелке</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g03">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>340</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G03 Круговая интерполяция против часовой стрелки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g90">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>390</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G90 Абсолютная система координат</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g91">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>G91 Относительная система координат</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_colebrate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Калибровка (переместиться на 0,0)</string>
      </property>
     </widget>
     <widget class="QLabel" name="mode_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear_img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить картинку</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="radiusLineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="radius_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
